internal/server/config.go

```diff
@@ -961,17 +465,12 @@ func (s Server) loadCredential(db data.WriteTxn, identity *models.Identity, pass
 	return nil
 }
 
-func (s Server) loadAccessKey(db data.WriteTxn, identity *models.Identity, key string) error {
+func (s Server) loadAccessKey(db data.WriteTxn, identity *models.Identity, key Secret) error {
```

Comment:
Secret is just a `type Secret string`. Passing `string` would be fine too, but `Secret` makes it a bit more explicit what kind of value is expected.

Maybe `Secret` should implement `String()` so that if it's accidentally printed it doesn't print the secret.

Reply:
I amended the latest commit and added `String` and `GoString` methods for `Secret`.